app/views/components/_markdown_editor.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= render "govuk_publishing_components/components/label", {
       text: label[:text],
```

Comment:
Would it work the same if we removed this line?

Reply:
yes, good point. removed them.